etcd-manager/test/integration/BUILD.bazel

```diff
@@ -31,6 +32,41 @@ go_test(
     ],
     deps = [
         "//pkg/apis/etcd",
+        "//pkg/backup",
+        "//pkg/etcd",
+        "//pkg/etcdversions",
+        "//test/integration/harness",
+        "//vendor/k8s.io/klog/v2:klog",
+    ],
+)
+
+go_test(
+    name = "integration_backuprestore_test",
+    size = "enormous",  # Takes more than 900 seconds to run on travis
```

Comment:
Would it make sense to move the test to a separate job similar to what we do for with `test-upgradedowngrade`?
https://github.com/kubernetes-sigs/etcdadm/blob/40001ec166bac2cf049ac2dd5ef63c68394aa70a/.github/workflows/main.yml#L41